joomla/joomla-cms
# [4.0] Remove Fade effect

## Summary of Changes
### Faster login
Removes the delay which is counter-logic

### Solves stutter bug #25904
https://github.com/joomla/joomla-cms/issues/25904#issuecomment-522082581


### Solves internet explorer bug #26005
(of course it wont work nicely with IE but at least someone could now log in if they have an emergency AND it opens the possibility to add a "does not work with IE" message" which you can't before this PR

### Testing Instructions
Apply PR and then run npm i

Check that login works
It should now be faster and smoother

Bonus try and open the admin with ie11 

(First of several PR on atum)
